load-agent/agent.ts

```diff
@@ -270,6 +270,15 @@ const pingMediator = async (agent) => {
   }
 }
 
+let deleteOobRecordById = async (agent, id) => {
+  try {
+    const resp = await agent.oob.deleteById(id);
+    
+  } catch (error) {
+    process.stderr.write('ERROR'+ '\n' + error + '\n')
```

Comment:
The format throughout `agent.ts` has been to throw the error directly. If you throw the error, it gets caught by this line: https://github.com/hyperledger/aries-akrida/blob/a327e041d60a221faa694e5d2d5891a866685454/load-agent/agent.ts#L566

I would suggest, rather than this parse, just `throw 'OOB Record not found!'`

Reply:
Taking this try/catch out to let it propagate up. 👍